policyengine_us/variables/gov/irs/income/taxable_income/adjusted_gross_income/irs_gross_income/social_security/taxable_social_security.py

```diff
@@ -11,16 +11,16 @@ class taxable_social_security(Variable):
     def formula(person, period, parameters):
         unit_tss = person.tax_unit("tax_unit_taxable_social_security", period)
         # allocate unit_tss to head and spouse in proportion to social_security
-        unit_socsec = person.tax_unit("tax_unit_social_security", period)
-        unit_has_socsec = unit_socsec > 0
         ind_socsec = person("social_security", period)
+        unit_socsec = person.tax_unit("tax_unit_social_security", period)
+        person_frac = np.zeros_like(unit_socsec)
```

Comment:
```suggestion
        # Compute each person's share of tax unit's Social Security benefits.
        # Use mask instead of where to avoid divide-by-zero warnings. Default to zero.
        person_frac = np.zeros_like(unit_socsec)
```